packages/syft/src/syft/service/job/job_stash.py

```diff
@@ -388,8 +388,12 @@ def wait(self):
         if self.resolved:
             return self.resolve
 
+        if subjobs:
+            self.result.wait()
+            return self.resolve
+
         print_warning = True
-        while True:
+        while not subjobs and True:
```

Comment:
maybe this part is obsolete, subjobs must be False if we reach this line

Reply:
Removed in [47a7cc2](https://github.com/OpenMined/PySyft/pull/8392/commits/47a7cc2106944691d3dd08dc830a73d897075af3)